pkg/manager/service.go

```diff
@@ -0,0 +1,73 @@
+package manager
+
+import (
+	"context"
+
+	"github.com/gogo/protobuf/types"
+	"go.uber.org/zap"
+
+	"github.com/criticalstack/e2d/pkg/client"
+	"github.com/criticalstack/e2d/pkg/e2db"
+	"github.com/criticalstack/e2d/pkg/log"
+	"github.com/criticalstack/e2d/pkg/manager/e2dpb"
+)
+
+type ManagerService struct {
+	m *Manager
+}
+
+func (s *ManagerService) Health(ctx context.Context, _ *types.Empty) (*e2dpb.HealthResponse, error) {
+	resp := &e2dpb.HealthResponse{
+		Status: "not great, bob",
```

Comment:
"I'm afraid there's been a catastrophic failure, we are uncertain if the situation is recoverable.  A single node remains online."
"Status report?"
"...not great, bob"